pkgs/applications/system/asusctl/default.nix

```diff
@@ -13,32 +13,30 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "asusctl";
-  version = "4.5.8";
+  version = "4.6.0";
 
   src = fetchFromGitLab {
     owner = "asus-linux";
     repo = "asusctl";
     rev = version;
-    hash = "sha256-6AitRpyLIq5by9/rXdIC8AChMVKZmR1Eo5GTo+DtGhc=";
+    hash = "sha256-QXRUi/CiJTG04Kv/p3PWVloPIguLyfFQoiY6tCQIF/M=";
   };
 
   cargoLock = {
-    lockFile = ./Cargo.lock;
+    lockFile = "${src}/Cargo.lock";
```

Comment:
That's IFD.

Reply:
Ah, okay, I'll revert that and update the vendored lockfile then.